.github/workflows/docker.yml

```diff
@@ -13,22 +13,14 @@ jobs:
     strategy:
       matrix:
         package: [auth, backend]
-        platform: [linux/amd64, linux/arm64]
         push:
           - ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') }}
-        exclude:
-          - platform: linux/arm64
-            push: false
     steps:
       - name: Docker meta
         id: meta
         uses: docker/metadata-action@v4
         with:
           images: ghcr.io/${{ github.repository_owner }}/rafiki-${{ matrix.package }}
-          tags: |
-            type=raw,enable={{is_default_branch}},value=latest
-            type=raw,enable=true,value={{ sha }}
-            type=raw,enable=true,value={{ refs }}
```

Comment:
Does the removal of this ensure that the image tag =  gh tag?

Reply:
yeah, the default behavior is to add the git tag as a label and the other defaults do what we were already doing (branch name, commit sha, etc)